scripts/launch-xrd

```diff
@@ -257,18 +246,15 @@ while [[ $# -gt 0 ]]; do
            shift
            ;;
         *)
-            if [[ ! $IMAGE ]]; then
-                IMAGE=$1
-                shift
-            else
-                echo "Unexpected argument '$1'" >&2
-                bad_usage
-            fi
-            ;;
+           CTR_MGR_ARGS+=("$1")
+           shift
+           ;;
     esac
 done
 
-if [[ ! $IMAGE ]]; then
+if [[ ${CTR_MGR_ARGS[-1]} ]]; then
+    IMAGE="${CTR_MGR_ARGS[-1]}"
```

Comment:
Could optionally save a line with
```bash
IMAGE="${CTR_MGR_ARGS[-1]}"
if [[ ! $IMAGE ]]; then
    echo "Must specify container image" >&2
    bad_usage
fi
```
Whatever looks nicer ;)

Reply:
I just made a change to check if the array is empty. Because from what I understand the old version and this will result in "bad array subscript" if the array is empty